### PR TITLE
Chapter 3,4,5

### PR DESCRIPTION
**Pull request by luctq-0138**
image

### CODE REVIEW

**Comment by luctq-0138** [comment]
Chapter 3: Mostly static page
Chapter 4: Rails-flavored Ruby
Chapter 5: Filling in the layout

**Comment by luctq-0138** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+      "data-turbolinks-track": "reload" %>
     <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <%= render "layouts/shim" %>
   </head>
 
   <body>
-    <%= yield %>
+    <%= render "layouts/header" %>
+    <%= link_to "English", locale:"en" %>
+    <%= link_to "Vietnamese", locale:"vi" %>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]:
```suggestion
    <%= link_to "English", locale: :en %>
    <%= link_to "Vietnamese", locale: :vi %>
```

**Review by ghost — COMMENTED** [review]
cmt nhỏ fix rồi merged em nhé

**Pull request merged**